index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="index.css">
+    <!-- Template 2-->
+</head>
+
+<body>
+    <header>
+        <div class="header_container">
+            <div class="item flex_one">
+                <h1 class="name">John Doe</h1>
+            </div>
+            <div class="item flex_two">
+                <nav>
+                    <div class="menu_bar">
+                        <img src="img/Union.png" alt="Menu bar">
+                    </div>
+                </nav>
+            </div>
+        </div>
+    </header>
+    <main class="main_container">
+        <section>
+            <h1>Hey There. I’m Farida</h1>
+            <h2>I’am a Software Developer</h2>
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+        </section>
+        <section class="social_media">
+            <img class="img img1" src="img/github.png" alt="GitHub">
+            <img class="img img2" src="img/linkin.png" alt="Linkedin">
+            <img class="img img3" src="img/yo.png" alt="Yo">
+            <img class="img img4" src="img/twitter.png" alt="Twitter">
+            <img class="img img5" src="img/Vector.png" alt="Vector">
```

Comment:
Kindly use `<ul><li><a>` elements for these social links. Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href. Also please check the alignment and margins are implemented well according to Figma template.